wrapper.h

```diff
@@ -43,3 +40,11 @@ typedef ptrdiff_t R_xlen_t_rust;
 #include <R_ext/GraphicsEngine.h>
 #include <R_ext/Applic.h>
 #include <R_ext/Random.h>
+
+
+/// <div rustbindgen replaces="Rcomplex"></div>
```

Comment:
Can you write a regular comment above this explaining why we re-define `Rcomplex`, perhaps with some links? Similar to what is written before `R_xlen_t`.